website/content/en/highlights/2021-08-24-vector-aggregator.md

```diff
@@ -0,0 +1,57 @@
+---
+date: "2021-08-24"
+title: "Moving the Aggregator chart to public beta"
+description: "The Vector Aggregator Helm chart is now publicly available"
+authors: ["spencergilbert"]
+pr_numbers: [8801]
+release: "0.16.0"
+hide_on_release_notes: false
+badges:
+  type: "announcement"
+  platforms: ["helm"]
+---
+
+We are happy to announce that our `vector-aggregator` chart is now publicly available. While
+we continue to iterate on and make improvements to the chart, we feel it is ready to get
+wider feedback from the community at large.
+
+We have also created a dedicated Discord channel, [#aggregator][discord], for support and questions.
```

Comment:
Maybe we could add a bit about what an aggregator is for people that aren't familiar? We can link to https://vector.dev/docs/setup/deployment/roles/#aggregator . I realize this is just announcing the chart's availability, but I can imagine a lot of users coming into this highlight having no idea what an "aggregator" is.

Should we also add or link to a description of what the chart does?